cmd/keycloakb/keycloak_bridge.go

```diff
@@ -109,9 +109,13 @@ func main() {
 	var c = config(log.With(logger, "unit", "config"))
 	var (
 		// Component
-		httpAddr                = c.GetString("component-http-host-port")
 		authorizationConfigFile = c.GetString("authorization-file")
 
+		// Publishing
+		httpAddrInternal = c.GetString("http-host-port-internal")
+		httpAddrManagement = c.GetString("http-host-port-management")
+		httpAddrAccount = c.GetString("http-host-port-account")
```

Comment:
For database, we are using prefixes (dbaudits-rw-host-port, dbaudits-ro-host-port, config-host-port)... I would see the same thing here : internal-http-host-port, management-http-host-port and account-http-host-port (this way, internal, management and account would be prefixes too for other parameters)

Reply:
Done